BookingsAPI/Bookings.IntegrationTests/Helper/TestDataManager.cs

```diff
@@ -60,45 +63,72 @@ public async Task<VideoHearing> SeedVideoHearing(Action<SeedVideoHearingOptions>
-            
-            var participant = videoHearing.AddIndividual(person1, claimantLipHearingRole, claimantCaseRole,
+
+           videoHearing.AddIndividual(person1, claimantLipHearingRole, claimantCaseRole,
                 $"{person1.FirstName} {person1.LastName}");
 
-            var participantSolicitor = videoHearing.AddSolicitor(person2, claimantSolicitorHearingRole, claimantCaseRole,
-                $"{person2.FirstName} {person2.LastName}", string.Empty, string.Empty);
-            
+            videoHearing.AddSolicitor(person2, claimantSolicitorHearingRole, claimantCaseRole,
+                $"{person2.FirstName} {person2.LastName}", string.Empty, "Ms X");
+
             videoHearing.AddSolicitor(person3, defendantSolicitorHearingRole, defendantCaseRole,
-                $"{person3.FirstName} {person3.LastName}", string.Empty, string.Empty);
+                $"{person3.FirstName} {person3.LastName}", string.Empty, "Ms Y");
 
             videoHearing.AddJudge(person4, judgeHearingRole, judgeCaseRole, $"{person4.FirstName} {person4.LastName}");
 
             videoHearing.AddCase("1234567890", "Test Case", true);
             videoHearing.AddCase("1234567891", "Test Case2", false);
 
-            if(addSuitabilityAnswer)
-            {
-                participant.AddSuitabilityAnswer("NEED_INTERPRETER", "No", "");
-                participant.AddSuitabilityAnswer("SUITABLE_ROOM_AVAILABLE", "Yes", "");
-                participant.UpdatedDate = DateTime.UtcNow;
-
-                participantSolicitor.AddSuitabilityAnswer("ABOUT_YOUR_CLIENT", "No", "");
-                participantSolicitor.AddSuitabilityAnswer("SUITABLE_ROOM_AVAILABLE", "No", "Comments");
-                participantSolicitor.UpdatedDate = DateTime.UtcNow;
-            }
-            
             using (var db = new BookingsDbContext(_dbContextOptions))
             {
                 await db.VideoHearings.AddAsync(videoHearing);
                 await db.SaveChangesAsync();
             }
-
             var hearing = await new GetHearingByIdQueryHandler(new BookingsDbContext(_dbContextOptions)).Handle(
+               new GetHearingByIdQuery(videoHearing.Id));
+            _individualId = hearing.Participants.FirstOrDefault(x => x.HearingRole.Name.ToLower().IndexOf("judge") < 0 && x.HearingRole.Name.ToLower().IndexOf("solicitor") < 0).Id;
+            _participantSolicitorIds = hearing.Participants.Where(x => x.HearingRole.Name.ToLower().IndexOf("solicitor") >= 0).Select(x => x.Id).ToList();
+
+            if (addSuitabilityAnswer)
+            {
+                await AddQuestionnaire();
+            }
+
+            hearing = await new GetHearingByIdQueryHandler(new BookingsDbContext(_dbContextOptions)).Handle(
                 new GetHearingByIdQuery(videoHearing.Id));
-            
             _seededHearings.Add(hearing.Id);
             return hearing;
         }
 
+        private async Task AddQuestionnaire()
```

Comment:
I think you could refactor this method to be a bit simpler and do less things. Maybe break it into two different methods?

`AddIndividualQuestionnaire(individualId)`
`AddRepresentativeQuestionnaire(representativeId)`

And then call them from outside:
```csharp
AddIndividualQuestionnaire(_individualId);
_participantSolicitorIds.ForEach(AddRepresentativeQuestionnaire);
```

Reply:
Done